controllers/common.go

```diff
@@ -21,6 +22,7 @@ const (
 	collectorDeploymentName = "search-collector"
 	indexerDeploymentName   = "search-indexer"
 	postgresDeploymentName  = "search-postgres"
+	AnnotationSearchPause   = "search-pause"
```

Comment:
How about "operator-pause" to keep it generic?
How does mch calls it? Is it mch-pause?

Reply:
yes - `mch-pause` .  Keeping same naming convention.